analytics_queries/schools.sql

```diff
@@ -7,14 +7,19 @@ SELECT DISTINCT s.urn,
                 icp.created_at                     as tutor_nominated_at,
                 (u.current_sign_in_at IS NOT NULL) as tutor_signed_in,
                 u.full_name                        as tutor_name,
-                u.email                            as tutor_email
+                u.email                            as tutor_email,
+                (pp.id IS NOT NULL)                as sit_mentor,
+
 FROM schools s
          LEFT OUTER JOIN school_cohorts sc on s.id = sc.school_id
-         LEFT OUTER JOIN cohorts c on sc.cohort_id = c.id
+         LEFT OUTER JOIN cohorts c on sc.cohort_id = c.id 1
```

Comment:
What is this for?

Reply:
I'd say me trying to switch buffer in vi. Removed.